sensor/common/sensor/sensor.go

```diff
@@ -208,7 +208,7 @@ func (s *Sensor) Start() {
 
 	for _, component := range s.components {
 		if err := component.Start(); err != nil {
-			log.Panicf("Sensor component %T failed to start: %v", component, err)
+			_ = utils.Should(errors.Wrapf(err, "Sensor component %T failed to start", component))
```

Comment:
```suggestion
			_ = utils.Should(errors.Wrapf(err, "sensor component %T failed to start", component))
```
https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
Thanks, applied